readme.md

```diff
@@ -19,7 +19,7 @@ todo
 ## Usage
 
 ```js
-todo
+Vbb get off bot is a tool that can be used to help remind you when to get off of the sub way. It does this by "..." and "..." 
```

Comment:
This is basically mentioned at the top of the readme. In this section, there should be a command to run the bot.

See also https://github.com/derhuerst/vbb-get-off-bot/pull/2/files#r222482612
https://github.com/derhuerst/vbb-get-off-bot/pull/2/files#r22248261
https://github.com/derhuerst/vbb-get-off-bot/pull/2/files#r22248261
https://github.com/derhuerst/vbb-get-off-bot/pull/2/files#r22248261222
https://github.com/derhuerst/vbb-get-off-bot/pull/2/files#r22248261
https://github.com/derhuerst/vbb-get-off-bot/pull/2/files#r22248261
https://github.com/derhuerst/vbb-get-off-bot/pull/2/files#r22248261222
https://github.com/derhuerst/vbb-get-off-bot/pull/2/files#r22248261
https://github.com/derhuerst/vbb-get-off-bot/pull/2/files#r2224826122